IBX-5450 Fixes to Commerce procedures in User Doc (#200)
ezsystems/user-documentation: docs/commerce/order_management/order_management.md

# Order management

> The Order management module allows users to review order details, track order completion status, and cancel orders that are created following purchases made by customers of the e-commerce presence.

Reviewer: ```suggestion
The Order management module allows users to review order details, track order completion status, and cancel orders that are created when customers purchase products in the e-commerce system.
```

> Depending on the permissions assigned to your [user role](permissions_and_users.md), you might be able to track and manage orders placed by all the store customers, or just your own ones.

Reviewer: ```suggestion
Depending on the permissions assigned to your [user Role](permissions_and_users.md), you might be able to track and manage orders placed by all the store customers, or just your own ones.
```
